From 8d76b69da060e5871facf17acf2ad4531a0a2713 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:25 -0400
Subject: [usb] ohci: kill ohci_vdbg

Message-id: <1410887463-20674-194-git-send-email-dzickus@redhat.com>
Patchwork-id: 95563
O-Subject: [RHEL7 PATCH 193/591] ohci: kill ohci_vdbg
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit d2c4254ff9753485900762dea98958ae4622aa05
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:22:58 2013 +0100

    ohci: kill ohci_vdbg

    With the introduction of dynamic debugging it has become redundant.
    Collapse it with ohci_dbg()

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ohci-hcd.c b/drivers/usb/host/ohci-hcd.c
index 13b8ccd..18cc500 100644
--- a/drivers/usb/host/ohci-hcd.c
+++ b/drivers/usb/host/ohci-hcd.c
@@ -832,7 +832,7 @@ static irqreturn_t ohci_irq (struct usb_hcd *hcd)
  }
 
  if (ints & OHCI_INTR_RHSC) {
-  ohci_vdbg(ohci, "rhsc\n");
+  ohci_dbg(ohci, "rhsc\n");
   ohci->next_statechange = jiffies + STATECHANGE_DELAY;
   ohci_writel(ohci, OHCI_INTR_RD | OHCI_INTR_RHSC,
     &regs->intrstatus);
@@ -854,7 +854,7 @@ static irqreturn_t ohci_irq (struct usb_hcd *hcd)
   * this might not happen.
   */
  else if (ints & OHCI_INTR_RD) {
-  ohci_vdbg(ohci, "resume detect\n");
+  ohci_dbg(ohci, "resume detect\n");
   ohci_writel(ohci, OHCI_INTR_RD, &regs->intrstatus);
   set_bit(HCD_FLAG_POLL_RH, &hcd->flags);
   if (ohci->autostop) {
diff --git a/drivers/usb/host/ohci-q.c b/drivers/usb/host/ohci-q.c
index 182eaa2..d4253e3 100644
--- a/drivers/usb/host/ohci-q.c
+++ b/drivers/usb/host/ohci-q.c
@@ -143,7 +143,7 @@ static void periodic_link (struct ohci_hcd *ohci, struct ed *ed)
 {
  unsigned i;
 
- ohci_vdbg (ohci, "link %sed %p branch %d [%dus.], interval %d\n",
+ ohci_dbg(ohci, "link %sed %p branch %d [%dus.], interval %d\n",
   (ed->hwINFO & cpu_to_hc32 (ohci, ED_ISO)) ? "iso " : "",
   ed, ed->branch, ed->load, ed->interval);
 
@@ -290,7 +290,7 @@ static void periodic_unlink (struct ohci_hcd *ohci, struct ed *ed)
  }
  ohci_to_hcd(ohci)->self.bandwidth_allocated -= ed->load / ed->interval;
 
- ohci_vdbg (ohci, "unlink %sed %p branch %d [%dus.], interval %d\n",
+ ohci_dbg(ohci, "unlink %sed %p branch %d [%dus.], interval %d\n",
   (ed->hwINFO & cpu_to_hc32 (ohci, ED_ISO)) ? "iso " : "",
   ed, ed->branch, ed->load, ed->interval);
 }
@@ -761,7 +761,7 @@ static int td_done(struct ohci_hcd *ohci, struct urb *urb, struct td *td)
   urb->iso_frame_desc [td->index].status = cc_to_error [cc];
 
   if (cc != TD_CC_NOERROR)
-   ohci_vdbg (ohci,
+   ohci_dbg(ohci,
     "urb %p iso td %p (%d) len %d cc %d\n",
     urb, td, 1 + td->index, dlen, cc);
 
@@ -793,7 +793,7 @@ static int td_done(struct ohci_hcd *ohci, struct urb *urb, struct td *td)
   }
 
   if (cc != TD_CC_NOERROR && cc < 0x0E)
-   ohci_vdbg (ohci,
+   ohci_dbg(ohci,
     "urb %p td %p (%d) cc %d, len=%d/%d\n",
     urb, td, 1 + td->index, cc,
     urb->actual_length,
diff --git a/drivers/usb/host/ohci.h b/drivers/usb/host/ohci.h
index ea02722..9250cad 100644
--- a/drivers/usb/host/ohci.h
+++ b/drivers/usb/host/ohci.h
@@ -482,12 +482,6 @@ static inline struct usb_hcd *ohci_to_hcd (const struct ohci_hcd *ohci)
 #define ohci_warn(ohci, fmt, args...) \
  dev_warn (ohci_to_hcd(ohci)->self.controller , fmt , ## args )
 
-#ifdef OHCI_VERBOSE_DEBUG
-# define ohci_vdbg ohci_dbg
-#else
-# define ohci_vdbg(ohci, fmt, args...) do { } while (0)
-#endif
-
 /*-------------------------------------------------------------------------*/
 
 /*
-- 
1.7.1